README.md

```diff
@@ -11,7 +11,7 @@
 
 This module provides automatic tracing for Node.js applications with Stackdriver Trace. [Stackdriver Trace](https://cloud.google.com/cloud-trace/) is a feature of [Google Cloud Platform](https://cloud.google.com/) that collects latency data (traces) from your applications and displays it in near real-time in the [Google Cloud Console][cloud-console].
 
-![Stackdriver Trace Overview](doc/images/cloud-trace-overview-page.png)
+<img src="/doc/images/cloud-trace-overview-page.png" alt="Stackdriver Trace Overview" />
```

Comment:
This can't be a relative url.  You need to use an absolute one for it work on npm :)

Reply:
I think it should work, check [this commit](https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/commit/71ae615a31ab5bc37db72975e018dfc210834922).

Reply:
It doesn't seem to work right now: https://www.npmjs.com/package/@google-cloud/debug-agent

Reply:
Could you fix the debug agent URLs as well?